modules/javafx.graphics/src/main/java/javafx/geometry/Point3D.java

```diff
@@ -410,10 +410,14 @@ public Point3D interpolate(Point3D endValue, double t) {
     }
 
     /**
-     * Returns a hash code value for the point.
-     * @return a hash code value for the point.
+     * Determines whether or not two objects are equal. Two instances of {@code Point3D}
+     * are equal if the values of their x, y, and z properties are equal.
+     * @param obj an object to be compared with this {@code Point3D}.
```

Comment:
We tend to put a new line before the tags to separate them from the description, not sure if it's enforced.

Reply:
Not strictly enforced, but adding a blank line does aid readability (of the source code...the generated docs don't care).

Reply:
If I were to add empty lines before JavaDoc tags such as @param etc, then for consistency I think I should do it for the whole class, where they are not present.

Note, on line 32: **// PENDING_DOC_REVIEW of this whole class**
Maybe we should first decide whether we are scrubbing all doc for the whole class, or making a minimal change to the obviously incorrect information. My feeling is now tending toward the latter.

It could very well be that I misunderstood the comment by nlisker.